src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -822,15 +827,15 @@ void Method()
         }
 
         [WorkItem(150480, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/150480")]
-        [WpfFact, Trait(Traits.Feature, Traits.Features.AutomaticCompletion)]
+        [WpfFact]
         public void NotDelegatedAfterOpenBraceObjectCreationExpression()
         {
             Test(@"class TestClass
 {
     void Method()
     {
-        var pet = new List<int> { };
-        $$
+        var pet = new List<int> 
+            $$
```

Comment:
```suggestion
        var pet = new List<int>();
        $$
```

Reply:
📝 I really like this as a test case; hoping it's not too hard to implement in the suggested modified form.